ui/src/LandingPage.js

```diff
@@ -164,6 +188,12 @@ class LandingPage extends Component {
   closePrintTemplateDialog = () => {
     this.setState({ createPrintTemplateDialogDisplayed: false });
   };
+  closeSmsTemplateDialog = () => {
+    this.setState({
+      createSmsTemplateDialogDisplayed: false,
+      createSmsTemplateError: "",
```

Comment:
probably don't need to reset this if you already reset it when you open the dialog